src/test/java/com/nedap/openehr/lsp/BasicTest.java

```diff
@@ -25,17 +26,17 @@ public class BasicTest extends LanguageServerTestBase {
     @Test
     public void testBasics() throws IOException {
 
-        openResource("test_archetype.adls");
```

Comment:
Since openResource is defined in https://github.com/nedap/archetype-languageserver/blob/master/src/test/java/com/nedap/openehr/lsp/LanguageServerTestBase.java#L36 , should that just accept the enum directly, instead of the filename?